File: QTD.SPA/src/app/_DtoModels/DIFSurveyImport/ValidateCSV_DIFSurveyEmployeeResponse_Results_VM.ts
import { ValidateCSV_Results_VM } from "@models/ImportCSV/ValidateCSV_Results_VM";
import { ImportDatum_DIFSurveyEmployeeResponse_VM } from "./ImportDatum_DIFSurveyEmployeeResponse_VM";

export class ValidateCSV_DIFSurveyEmployeeResponse_Results_VM extends ValidateCSV_Results_VM{
  data: ImportDatum_DIFSurveyEmployeeResponse_VM[];
}

File: QTD2.Infrastructure/Model/DIFSurveyEmployeeResponse/ValidateCSV_DIFSurveyEmployeeResponse_Results_VM.cs
using QTD2.Infrastructure.Model.Import_CSV_VM;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.DIFSurveyEmployeeResponse
{
   public class ValidateCSV_DIFSurveyEmployeeResponse_Results_VM : ValidateCSV_Results_VM
    {
        public List<ImportDatum_DIFSurveyEmployeeResponse_VM> Data { get; set; }
    }
}
